app/mappers/form/work_contributors_mapper.rb

```diff
@@ -31,14 +31,31 @@ def call
           'suborganization_name' => suborganization_name,
           'stanford_degree_granting_institution' => stanford_degree_granting_institution?,
           'orcid' => orcid,
-          'with_orcid' => orcid.present?
+          'with_orcid' => orcid.present?,
+          'affiliations_attributes' => affiliations
         }
       end
 
       private
 
       attr_reader :contributor
 
+      def affiliations
```

Comment:
This is returning an array of hash(es), I'm curious if it should be returning a form perhaps...